parcels/interaction/distance_utils.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def fast_distance(lat1, lon1, lat2, lon2):
+    '''Compute the arc distance assuming the earth is a sphere.'''
+    g = np.sin(lat1)*np.sin(lat2)+np.cos(lat1)*np.cos(lat2)*np.cos(lon1-lon2)
+    return np.arccos(np.minimum(1, g))
```

Comment:
please write the source for this equation in the docstring

Reply:
Done